protocol/messenger.go

```diff
@@ -623,6 +625,7 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 
 	m.handleEncryptionLayerSubscriptions(subscriptions)
 	m.handleCommunitiesSubscription(m.communitiesManager.Subscribe())
+	m.handleCommunitiesHistoryArchivesSubscription(m.communitiesManager.Subscribe())
```

Comment:
I'm reusing the existing community manager subscriptions to signal community history events as well